snippets/base/migrations/0002_auto_20160215_1300.py

```diff
@@ -0,0 +1,21 @@
+# -*- coding: utf-8 -*-
+from __future__ import unicode_literals
+
+from django.db import migrations, models
+import snippets.base.fields
+import snippets.base.models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('base', '0001_initial'),
+    ]
+
+    operations = [
+        migrations.AlterField(
+            model_name='uploadedfile',
+            name='file',
+            field=models.FileField(upload_to=snippets.base.models._generate_filename),
```

Comment:
This migration doesn't appear to actually do anything at the db level, it just removes the `storage` kwarg present in the `0001_initial` migration. It seems safe, but we should validate that it works with `settings.DEFAULT = 'storages.backends.overwrite.OverwriteStorage'` in the legacy staging environment-- not a blocker for merge, just noting here.